plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-taxes.php

```diff
@@ -102,10 +126,9 @@ public function test_get_reports() {
 		$tax_report = reset( $reports );
 
 		$this->assertEquals( 1, $tax_report['tax_rate_id'] );
-		$this->assertEquals( 'TestTax', $tax_report['name'] );
 		$this->assertEquals( 7, $tax_report['tax_rate'] );
 		$this->assertEquals( 'US', $tax_report['country'] );
-		$this->assertEquals( 'GA', $tax_report['state'] );
+		$this->assertEquals( 'NY', $tax_report['state'] );
```

Comment:
Any reason for this change? I mean, I'm partial to NY 😄, but there doesn't seem to be a need for this change.